independent-projects/bootstrap/core/src/main/java/io/quarkus/bootstrap/app/QuarkusBootstrap.java

```diff
@@ -127,8 +125,7 @@ public CuratedApplication bootstrap() throws BootstrapException {
                 .setAppArtifact(appArtifact)
                 .setManagingProject(managingProject)
                 .setForcedDependencies(forcedDependencies)
-                .setProjectRoot(getProjectRoot() != null ? getProjectRoot()
-                        : getApplicationRoot());
+                .setProjectRoot(getProjectRoot());
```

Comment:
I'll check whether this works for the platform TS. There is still a bit of a mess around these.

Reply:
After this PR I want to get rid of the whole ProjectRoot thing and replace it with a more general MetadataSource concept. Getting IDE run to work in gradle is going to take some work, and this refactoring will be part of that. Lets discuss on zulip when you are around.